README.md

```diff
@@ -96,7 +96,7 @@ From `Android Studio`, download and install the desired API Level for the SDK. T
 * Launch `Android Studio` and accept all defaults.
 * Click on the SDK Manager Icon in the toolbar.
 * Navigate to `Appearance & Behavior` -> `System Settings` -> `Android SDK`
-* Select desired API Levels (select all >= 23).
+* Select Android 6.0 (Marshmallow) (API Level 23).
```

Comment:
I'm not super confident on this specific change. Why is 23 explicitly required? Why did it previous imply that you need multiple versions of the SDK? I was successful installing just the latest two versions, skipping 23 and 24.

Reply:
I believe API Level 25 is the only one that is actually required for this journey. Can anyone confirm?

Reply:
@dolph Since we wanted to cover majority of the phones that didn't have the latest version!

Reply:
Yes, only one version is required, but it has to be supported by the android device.